.jshintrc

```diff
@@ -6,17 +6,10 @@
   "futurehostile": true,
   "latedef": "nofunc",
   "noarg": true,
+  "node": true,
   "nonew": true,
   "shadow": false,
   "strict": false,
   "undef": true,
-  "globals": {
```

Comment:
These globals were only ever used by generated sync functions, so they've been moved to a more appropriate location.